nodes/containers/nodes-containers-using.adoc

```diff
@@ -40,9 +39,38 @@ To avoid losing containers due to kernel memory issues, ensure that the containe
 $(nproc) X 1/2 MiB
 ----
 
+[id="nodes-containers-runtimes"]
+== About the container engine and container runtime
 
+A _container engine_ is a piece of software that processes user requests, including command line options and image pulls. The container engine uses a _container runtime_ (also called a _lower-level container runtime_) to run and manage the components required to deploy and operate containers. You likely will not need to interact with the container engine or container runtime.  
+
+[NOTE]
+====
+The {product-title} documentation uses the term _container runtime_ to refer to the lower-level container runtime. Other documentation can refer to the container engine as the container runtime.
+====
+
+{product-title} uses CRI-O as the container engine and runC or crun as the container runtime. The default container runtime is runC. Both container runtimes adhere to the link:https://www.opencontainers.org/[Open Container Initiative (OCI)] runtime specifications. 
+
+include::snippets/about-crio-snippet.adoc[]
+
+runC, developed by Docker and maintained by the Open Container Project, is a lightweight, portable container runtime written in Go. crun, developed by Red Hat, is a fast and low-memory container runtime fully written in C. As of {product-title} {product-version}, you can select between the two.
+
+:FeatureName: crun container runtime support
+include::snippets/technology-preview.adoc[]
+
+crun has several improvements over runc, including:
+
+* a smaller binary
+* quicker processing
+* a lower memory footprint
+
+runC has some benefits over crun, including:
+
+* runC is the most popular OCI container runtime.
+* runC has a longer tenure in production.
+* runC is the default container runtime of CRI-O.
```

Comment:
```suggestion
* Default container runtime of CRI-O
```

Reply:
Thanks for the tips. I must have re-written those lists 6 times. Still doesn't sound right to me for some reason. But I'll take your advice.